examples/call/sui/sources/connected.move

```diff
@@ -0,0 +1,60 @@
+module connected::connected;
+
+use connected::cetusmock::{GlobalConfig, Partner, Pool, Clock, swap_a2b};
+use gateway::gateway::Gateway;
+use std::ascii::String;
+use sui::address::from_bytes;
+use sui::coin::Coin;
+use sui::event;
+use sui::tx_context;
+
+// Withdraw and Call Example
+
+public entry fun on_call<SOURCE_COIN, TARGET_COIN>(
+    in_coins: Coin<SOURCE_COIN>,
+    cetus_config: &GlobalConfig,
+    pool: &mut Pool<SOURCE_COIN, TARGET_COIN>,
+    cetus_partner: &mut Partner,
+    clock: &Clock,
+    data: vector<u8>,
+    ctx: &mut TxContext,
+) {
+    let coins_out = swap_a2b<SOURCE_COIN, TARGET_COIN>(
```

Comment:
We can maybe give some more info that this a call that replicate the exact interface as Cetus?

Reply:
dcc372d